### ISSUE TITLE
Support dependency versioning for R

### ISSUE THREAD

**Issue by jmlord** [GEO-BON/bon-in-a-box-pipeline-engine#24]
Dependency management should be officially supported by the pipeline engine, and properly documented.

**Comment by jmlord** [comment]
@glaroc worked on this on branch conda.

- [ ] We now need to add in the script yaml a conda section where one can specify an environment that differs from the base. 

```
name: environment-name
channels:
  - conda-forge
dependencies:
  - package

```

- [ ] If not present, "rbase" environment is used.
- [ ] GitHub packages are temporarily added to the runner.
- [ ] Save the environment to a text file in the script output folder.

**Comment by jmlord** [comment]
Closed by #116
